change/@fluentui-react-badge-e7a2177f-7d2a-4f6e-9312-074b3a71b06d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29288&commitId=ea8969d3b53fa5d1fe6b4b38dc09f9833bffbd3d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.customSize+name+badge.normal.chromium.png|2| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|2| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|28| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|14| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|29| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|8| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|30| Changed|

</details>


  <details><summary><b>Persona Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.basic - High Contrast.normal.chromium.png|4| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|2| Changed|
|Persona Converged.basic - Dark Mode.normal.chromium.png|1| Changed|
|Persona Converged.basic.normal.chromium.png|2| Changed|
|Persona Converged.textPosition.normal.chromium.png|6| Changed|
|Persona Converged.textAlignment.normal.chromium.png|4| Changed|
|Persona Converged.size-presence.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - OOF status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|83| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|83| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - inverted background</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - inverted background.default.default.chromium.png|446| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.default.chromium.png|381| Changed|
|PresenceBadge Converged - inverted background.default - High Contrast.default.chromium.png|356| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - sizes</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|5| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|5| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - status.default.chromium.png|43| Changed|
|PresenceBadge Converged - status.default - RTL.chromium.png|43| Changed|

</details>


  <details><summary><b>Table layout flex</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|9| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|

</details>


  <details><summary><b>Table layout flex - cell actions</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|

</details>


  <details><summary><b>Table layout table</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|9| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|

</details>


  <details><summary><b>Table layout table - cell actions</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>